.organizationmap

```diff
@@ -222,7 +225,9 @@ Intel <contact@intel.com> Yuan Zhou <yuan.zhou@intel.com>
 Intel <contact@intel.com> Zhiqiang Wang <zhiqiang.wang@intel.com>
 inwinSTACK inc. <leo.l@inwinstack.com> Chang-Yi Lee <cy.l@inwinstack.com>
 inwinSTACK inc. <leo.l@inwinstack.com> Albert Tu <albert.t@inwinstack.com>
+inwinSTACK inc. <leo.l@inwinstack.com> MinSheng Lin <minsheng.l@inwinstack.com>
 IO <contact@io.com> Joe Julian <jjulian@io.com>
+iQIYI <edison@iqiyi.com> Chengwei Yang <yangchengwei@qiyi.com>
```

Comment:
Le 07/09/16 à 11:52, Chengwei Yang a écrit :
>
> In .organizationmap 
> <https://github.com/ceph/ceph/pull/10955#discussion_r77792412>:
>
> >  IO <contact@io.com> Joe Julian <jjulian@io.com>
> > +iQIYI <edison@iqiyi.com> Chengwei Yang <yangchengwei@qiyi.com>
>
> Oh, sorry, the domain name is |qiyi.com|, not |iqiyi.com|, :-)
>

HI,
My fault, corrected.
Is 
https://github.com/ceph/ceph/pull/10955/commits/aa40a73d028f82d9f05e40a002a7d7c827f4f770 
correct ?

Cheers,